packages/oas-utils/src/helpers/httpMethods.ts

```diff
@@ -1,42 +1,21 @@
 import type { RequestMethod } from '@/entities/spec/requests'
 
+/**
+ * HTTP methods in a specific order
+ * Do not change the order
+ */
```

Comment:
changing order in requests.ts won't impact the order to i changed it here + comment and kept previous code along @amritk

Reply:
yes we kinda do as we add properties per method, but could be refactored